metadata.json

```diff
@@ -9,8 +9,9 @@
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
+    {"name":"puppet/tea","version_requirement":"= 0.2.1"}
```

Comment:
This may be too strict of a requirement. It doesn't allow room for tea to fix bugs or add backwards compatible features without forcing an upgrade of the NTP module.